docs/public/installation/zonemaster-backend.md

```diff
@@ -711,7 +715,53 @@ Now go back to "[Database configuration](#53-database-configuration-freebsd)"
 to create the database tables and then continue with the steps after that.
 
 
-## 9. Cleaning up the database
+## 9. Installation with Clickhouse (experimental)
+
+> This is an experimental feature.
+
+First follow the installation instructions for the OS in question, and then go
+to this section to install Clickhouse.
+
+To install Clickhouse, follow the instructions from the [official
+documentation](https://clickhouse.com/docs/en/install#available-installation-options).
+
+Install the Perl bindings (Clickhouse relies on the MySQL DBI):
+
+* Rocky Linux: `sudo dnf install -y perl-DBD-MySQL`
+* Debian/Ubuntu: `sudo apt install libdbd-mysql-perl`
+* FreeBSD: `pkg install mysql57-server p5-DBD-mysql`
+
+To create the database and the database user (unless you keep an old database).
+Edit the command first if you want a non-default database name, user name or
+password.
+
+> **Note**: The Clickhouse MySQL interface requires a [double SHA1
+> password](https://clickhouse.com/docs/en/operations/settings/settings-users#password_double_sha1_hex)
+> ```
+> PASSWORD=$(base64 < /dev/urandom | head -c8); echo "$PASSWORD"; echo -n "$PASSWORD" | sha1sum | tr -d '-' | xxd -r -p | sha1sum | tr -d '-'
+> ```
+
+```sh
+clickhouse-client --ask-password -q "CREATE USER zonemaster IDENTIFIED WITH DOUBLE_SHA1_HASH BY '<DOUBLE_SHA1_PASSWORD>';"
+clickhouse-client --ask-password -q "CREATE DATABASE zonemaster;"
+clickhouse-client --ask-password -q "GRANT CREATE TABLE, DROP TABLE, SELECT, INSERT, ALTER UPDATE ON zonemaster.* TO zonemaster;"
+```
+
+Update the `backend_config.ini` file with the correct databse engine, database
+name, username and password if non-default values are used. The file location
+differs based on the used OS:
+
+* Rocky Linux: `/etc/zonemaster/backend_config.ini`
+* Debian/Ubuntu: `/etc/zonemaster/backend_config.ini`
+* FreeBSD: `/usr/local/etc/zonemaster/backend_config.ini`
+
+> **Note:** See the [backend configuration] documentation for details.
+
+Now go back to "[Database configuration](#43-database-configuration-debianubuntu)"
+to create the database tables and then continue with the steps after that.

```

Comment:
The link is for Debian/Ubuntu only. Please update to
```
Now go back to "Database configuration" to create the database tables and then continue with the steps after
that. Select your OS:
* [Debian/Ubuntu](#43-database-configuration-debianubuntu)
* [FreeBSD](#53-database-configuration-freebsd)
* [Rocky Linux](#33-database-configuration-rocky-linux)
```

Reply:
Suggestion applied.